sigs.yaml

```diff
@@ -2372,6 +2372,14 @@ sigs:
     url: https://zoom.us/j/841218129
     archive_url: https://docs.google.com/document/d/13mwye7nvrmV11q9_Eg77z-1w3X7Q1GTbslpml4J7F3A/edit
     recordings_url: https://www.youtube.com/watch?v=PweKj6SU7UA&list=PL69nYSiGNLP2vwzcCOhxrL3JVBc-eaJWI
+  - description: descheduler biweekly meeting
+    day: Tuesday
+    time: "17:00"
```

Comment:
could I suggest another time like `18:00` (2pm et) or `15:00` (11am et)? I'd like to be able to attend these